packager/react-packager/src/SocketInterface/index.js

```diff
@@ -35,10 +35,15 @@ const SocketInterface = {
         }
       });
 
-      const sockPath = path.join(
+      let sockPath = path.join(
```

Comment:
cannot use const, the variable is assigned another value in lines 43-44. I thought let would be safest (smallest scope)